code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -306,7 +303,6 @@
 	name = "medical hardsuit"
 	desc = "A special suit that protects against hazardous, low pressure environments. Built with lightweight materials for easier movement."
 	item_state = "medical_hardsuit"
-	slowdown = 1
```

Comment:
maybe we should consider adding some life support shit into medical hardsuits like maximum disease protection and shit